_posts/2016/2016-11-28-xp-days-germany-2016.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "XP Days Germany 2016"
+date: "2016-11-28 12:11:00"
+image: blog-header/hamburg-harbor-city.jpg
+categories: events
+authors: ["Benjamin", "Bastian"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+The [13th XP Days Germany](http://www.xpdays.de/2016/) took place again at the [Handwerkskammer Hamburg](https://www.hwk-hamburg.de) – within a stone's throw of [ePages HQ](http://www.epages.com).
+XP Day(s) are a global conference series for developers and by developers with events in France, England, Scotland, Benelux Union, Ukraine and even Vietnam.
+The integral part is the strong focus on agile software development as well as the values, principles and techniques of [eXtreme Programming](https://en.wikipedia.org/wiki/Extreme_programming).
+In this context also several product owners, scrum masters and innovation evangelists had the courage to participate and contribute.
+
+# The schedule
+
+This time the three-day-conference with around 170 participants was even accompanied by a [Devoxx4Kids](http://www.devoxx4kids-hamburg.de) event, so that theoretically the whole geek family could join.
+[Sunday](http://www.xpdays.de/2016/downloads/community-day/OpenSpaceSonntag.pdf) was assembled in an open space format with a session marketplace in the morning and after the lunch break.
+The sessions varied from solving coding katas to sharing knowledge and exchanging experiences.
+Monday and Tuesday in contrast provided a regular [conference schedule](http://www.xpdays.de/2016/programm/) with over 32 talks, 8 workshops, 11 lightning talks, 2 keynotes and some flexible slots – [Lean Coffee](http://german.leancoffee.org/) and an additional smaller open space.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Keynote-Feeling bei den XP Days <a href="https://twitter.com/hashtag/developer?src=hash">#developer</a> <a href="https://twitter.com/hashtag/agile?src=hash">#agile</a> <a href="https://twitter.com/hashtag/programming?src=hash">#programming</a> <a href="https://twitter.com/hashtag/conference?src=hash">#conference</a> <a href="https://twitter.com/hashtag/xdde16?src=hash">#xdde16</a> <a href="https://twitter.com/xd_de">@xd_de</a> <a href="https://t.co/wt0L6pnJyC">pic.twitter.com/wt0L6pnJyC</a></p>&mdash; Benjamin Nothdurft (@dataduke) <a href="https://twitter.com/dataduke/status/800710674884141056">November 21, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Our Docker talk and workshop
+
+After a short pitch in the opening talk we could gather more than 30 interested conference visitors to join our session: "[Welcome to Dockerland: Best Practises for Kickstarters and Tools by Example](http://www.xpdays.de/2016/sessions/116-welcome-to-dockerland-best-practices-for-kickstarters-tools-by-example)".
```

Comment:
a comma after opening talk is missing: "After a short pitch in the opening talk, we could..."